test/TestApp.AspNetCore.2.1/TestApp.AspNetCore.2.1.csproj

```diff
@@ -12,8 +12,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.App" />
-    <PackageReference Include="Microsoft.AspNetCore.Razor.Design" Version="2.1.2" PrivateAssets="All" />
+    <PackageReference Include="Microsoft.AspNetCore.App" Version="2.1.1" />
```

Comment:
@CodeBlanch , any reason why we need to add this? We are receiving some warnings after this change.

Reply:
Dang. I had to add that to get the W3C tests to pass on 2.1. Without the version they complain about no lower bound version or something? If you take this out in the 2 spots and then run the 2.1 docker-compose for W3C you should see the error.

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet/actions/runs/260622003

Reply:
Updated

Reply:
i saw ur hack! 👍